README.md

```diff
@@ -916,6 +920,14 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 ---
 
+## Regular Expressions
```

Comment:
Please add before this the "Back to Index" link. Refer to the [contribution guidelines](https://github.com/shahednasser/awesome-resources/blob/master/CONTRIBUTING.md) for more details.